### PR TITLE
Increase FinalityDepth & HistoryDepth defaults to match Polygon mainnet

### PR DESCRIPTION
**Pull request by reductionista**
We have verified that finality tags on zkevm polygon are just as far behind latest as they are on other polygon chains (often 300-500 blocks). HistoryDepth needs to be larger for the same reason, and allow room for uncles.

### CODE REVIEW

**Comment by jmank88** [comment]
You'll need to run `make config-docs` to pick up these default changes.

**Review by prashantkumar1982 — DISMISSED** [review]

**Review by simsonraj — APPROVED** [review]

File: docs/CONFIG.md [hunk 1]
```diff
@@ -3993,13 +3993,13 @@ TipCapMin = '1 wei'
 
 [GasEstimator.BlockHistory]
 BatchSize = 25
-BlockHistorySize = 12
+BlockHistorySize = 1000
```
Comment (patrickhuie19) [inline docs/CONFIG.md:3996]: Why increase the number of blocks used to calculate gas?

Comment (reductionista) [reply docs/CONFIG.md:3996]: That was an accidental change I made at some point yesterday and rolled back soon after. It's not changed in the actual defaults, must be that I forgot to regenerate the config?

Comment (reductionista) [reply docs/CONFIG.md]: @patrickhuie19 Thanks for catch, I've re-run "make config-docs" which removed those changes.  Approve?

**Review by prashantkumar1982 — APPROVED** [review]

File: docs/CONFIG.md (hunk 1 above)
Comment (patrickhuie19) [reply docs/CONFIG.md]: Sorry just saw this, but LGTM! - my github notifications are not surfacing replies to comments :/

**Pull request merged**